manifests/f/Ferdium/Ferdium/Nightly/6.2.1-beta.1/Ferdium.Ferdium.Nightly.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Ferdium.Ferdium.Nightly

```

```suggestion
PackageIdentifier: Ferdium.Ferdium.Beta
```
This should be under the beta folder, not the nightly folder